src/utils/components/ExpandSectionWithSwitch/ExpandSectionWithSwitch.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, FormEvent, ReactNode } from 'react';
+
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import { PopoverPosition, Split, SplitItem, Switch } from '@patternfly/react-core';
+
+import ExpandSection from '../../../views/clusteroverview/SettingsTab/ExpandSection/ExpandSection';
+
+import './ExpandSectionWithSwitch.scss';
+
+type ExpandSectionWithSwitchProps = {
+  children: ReactNode;
+  helpTextIconContent?: ReactNode;
+  isDisabled?: boolean;
+  switchIsOn: boolean;
+  toggleContent?: ReactNode;
+  toggleText?: string;
+  turnOnSwitch: (checked: boolean, event: FormEvent<HTMLInputElement>) => void;
+};
+
+const ExpandSectionWithSwitch: FC<ExpandSectionWithSwitchProps> = ({
+  children,
+  helpTextIconContent,
+  isDisabled,
+  switchIsOn,
+  toggleContent,
+  toggleText,
+  turnOnSwitch,
+}) => {
+  return (
```

Comment:
I think you don't need `return` here, you can omit it.

Reply:
Done!